### ISSUE TITLE
[Usable – Make Code – Language] There is no chevron or underline for multiple link controls under 'Select Language' pop-up window. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3661]
**User Experience:**
User will be misguided If any actionable control(link) doesn't have underline or chevron as they will skip the functionality of such link. Not only this, user will face disappointment if visually appearing link display as text as user won't be proceed further for relevant action.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Language " option and select it.
7. Navigate through all the controls appearing under "Select Language" dialog.
8. Verify visual all 'language link controls' and 'How do I add new language link' by using Developer tool(F12).

**Actual Result:**
There is no chevron or underline for 'language link controls' and 'How do I add new language link' present on 'Select language' pop-up window and due to this, user is getting affected as he/she might consider that link as text and will skip the functionality of such link.

**Expected Result:**
There must be chevron or underline on visual 'language link controls' and 'How do I add new language link' present on 'Select language' pop-up window, so that user could easily navigate that link to get more details.

**Note:**
1. This issue also exists with Firefox and Chrome Browser.
2. Same issue exists on 'Extensions' screen.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev4; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #Win10-Edge(Chromium)

![Usable_Language_Link does not have underline](https://user-images.githubusercontent.com/72871089/98900397-b1884b00-24d7-11eb-9bf0-19293483a191.jpg)
